zstacklib/zstacklib/utils/gpu.py

```
@@ -1,9 +1,11 @@
-import os
-
 from zstacklib.utils import log, linux
 from zstacklib.utils.bash import *
+
 import json
 
+from zstacklib.utils import log
```

Ruff (0.12.2)

Redefinition of unused `log` from line 1

Remove definition: `log`

(F811)